src/Sylius/Behat/Page/Admin/Order/ShowPage.php

```diff
@@ -340,6 +341,17 @@ public function getRouteName()
     /**
      * {@inheritdoc}
      */
+    public function hasNote($note)
+    {
+        $orderNotesElement = $this->getElement('order_notes');
+        Assert::notNull($orderNotesElement, 'Cannot find order notes');
```

Comment:
IIRC `getElement('order_notes')` throws exception when the element is not found.